tests/performance/dispatch/dispatch.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'performance_test'
+require 'app_generator/search_app'
+require 'performance/fbench'
+require 'environment'
+
+class DispatchMerge < PerformanceTest
+
+  def initialize(*args)
+    super(*args)
+  end
+
+  def setup
+    super
+    set_owner("balder")
+  end
+
+  def get_app()
+    SearchApp.new.sd(selfdir + "test.sd").search_dir(selfdir + "app").elastic.num_parts(8).redundancy(1).ready_copies(1)
+  end
+
+  def test_merge
+    set_description("Test merge speed with 8 backend and large offsets.")
+    @graphs = [
+      {
+        :title => "Qps",
+        :x => 'legend',
+        :y => 'qps',
+        :historic => true
+      },
+      {
+        :title => 'Average latency',
+        :x => 'legend',
+        :y => 'latency',
+        :historic => true
+      },
+      {
+        :x => 'legend',
+        :y => 'cpuutil',
+        :historic => true
+      },
+      {
+        :x => 'legend',
+        :y => 'memory.rss',
+        :historic => true
+      }
+    ]
+    num_docs = 100000
+    deploy_app(get_app())
+    container = (vespa.qrserver["0"] or vespa.container.values.first)
+    container.execute("g++ -Wl,-rpath,#{Environment.instance.vespa_home}/lib64/ -g -O3 -o #{dirs.tmpdir}/docs #{selfdir}/docs.cpp")
```

Comment:
Consider generating documents in Ruby instead. Any specific reason for using a C program for that purpose?